src/app/pages/uploads/upload-image/upload-image.component.scss

```diff
@@ -0,0 +1,178 @@
+@import "../../../shared-styles/all-sass-imports.scss";
+
+.upload-image {
+  width: 100%;
+  height: 100vh;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+
+  svg {
+    cursor: pointer;
+  }
+
+  header {
+    background-color: $primary_white-smoke;
+    color: #000;
+    padding: 20px 0;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .container {
+      width: 90%;
+      margin: 0 auto;
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+
+      .logo {
+        font-size: 32px;
+        font-weight: bold;
+        cursor: pointer;
+      }
+
+      h1 {
+        font-size: 20px;
+        color: $primary_color;
+      }
+    }
+  }
+
+  .content {
+    display: flex;
+    align-items: start;
+    justify-content: space-between;
+    flex: 1;
+    padding: 20px 100px;
+
+    form {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: 500px;
+      row-gap: 10px;
```

Comment:
`rem` units are more responsive than `px` and you can extract your most used styles into mixins and do `@include flex-col` or `@include flex-row`